sql/core/src/main/java/org/apache/spark/sql/execution/datasources/parquet/VectorizedColumnReader.java

```diff
@@ -174,6 +177,30 @@ private boolean isLazyDecodingSupported(PrimitiveType.PrimitiveTypeName typeName
     return isSupported;
   }
 
+  static int rebaseDays(int julianDays, final boolean failIfRebase) {
```

Comment:
nit but `failIfRebase` -> `shouldFailIfRebase`?

Reply:
it doesn't make a big difference... and shorter names are better.